projects/1.16.1/assets/ensorcellation/ensorcellation/lang/zh_cn.json

```
@@ -0,0 +1,112 @@
+{
+  "enchantment.ensorcellation.air_affinity": "空中速掘",
+  "enchantment.ensorcellation.air_affinity.desc": "在空中时的采掘速度不受影响。",
+
+  "enchantment.ensorcellation.angler": "钓者之赏",
+  "enchantment.ensorcellation.angler.desc": "钓鱼时有几率得到额外物品。",
+
+  "enchantment.ensorcellation.bulwark": "壁垒",
+  "enchantment.ensorcellation.bulwark.desc": "防止使用者被击退并禁用攻击者的武器。",
+
+  "enchantment.ensorcellation.cavalier": "骑士",
+  "enchantment.ensorcellation.cavalier.desc": "增加骑乘实体时的攻击伤害。",
+
+  "enchantment.ensorcellation.curse_fool": "愚钝诅咒",
+  "enchantment.ensorcellation.curse_fool.desc": "使用带有此魔咒的物品时阻止经验的获得。",
+
+  "enchantment.ensorcellation.curse_mercy": "仁慈诅咒",
+  "enchantment.ensorcellation.curse_mercy.desc": "阻止武器造成致命一击。",
+
+  "enchantment.ensorcellation.damage_ender": "末影杀手",
+  "enchantment.ensorcellation.damage_ender.desc": "提升对末影生物的伤害并阻止传送。",
+
+  "enchantment.ensorcellation.damage_illager": "灾厄杀手",
+  "enchantment.ensorcellation.damage_illager.desc": "提升对灾厄生物的伤害。",
+
+  "enchantment.ensorcellation.damage_villager": "村庄杀手",
+  "enchantment.ensorcellation.damage_villager.desc": "提升对村庄生物的伤害。",
+
+  "enchantment.ensorcellation.displacement": "移位",
+  "enchantment.ensorcellation.displacement.desc": "将攻击者随机传送走。",
+
+  "enchantment.ensorcellation.excavating": "范围挖掘",
+  "enchantment.ensorcellation.excavating.desc": "额外采集半径内的方块。",
+
+  "enchantment.ensorcellation.exp_boost": "洞察",
+  "enchantment.ensorcellation.exp_boost.desc": "增加使用带有此魔咒的物品时获得的经验。",
+
+  "enchantment.ensorcellation.fire_rebuke": "炽焰之斥",
+  "enchantment.ensorcellation.fire_rebuke.desc": "击退的同时点燃攻击者。",
+
+  "enchantment.ensorcellation.frost_aspect": "冰霜附加",
+  "enchantment.ensorcellation.frost_aspect.desc": "减缓并弱化目标。",
+
+  "enchantment.ensorcellation.frost_rebuke": "霜冻之斥",
+  "enchantment.ensorcellation.frost_rebuke.desc": "击退的同时减缓并弱化攻击者。",
+
+  "enchantment.ensorcellation.frost_walker": "冰霜行者",
+  "enchantment.ensorcellation.frost_walker.desc": "冻结脚下的水和熔岩，以便穿过。",
+
+  "enchantment.ensorcellation.furrowing": "直线耕犁",
+  "enchantment.ensorcellation.furrowing.desc": "将一条线上的方块转化为耕地。",
+
+  "enchantment.ensorcellation.gourmand": "美食家",
+  "enchantment.ensorcellation.gourmand.desc": "使吃下的食物更加滋补和可口。",
+
+  "enchantment.ensorcellation.hunter": "狩者之赏",
+  "enchantment.ensorcellation.hunter.desc": "杀死动物时有几率得到额外掉落物。",
+
+  "enchantment.ensorcellation.instigating": "煽动",
+  "enchantment.ensorcellation.instigating.desc": "对满生命值目标造成额外伤害。",
+
+  "enchantment.ensorcellation.leech": "吸血",
+  "enchantment.ensorcellation.leech.desc": "当杀死生物时回复生命值。",
+
+  "enchantment.ensorcellation.magic_edge": "魔法之刃",
+  "enchantment.ensorcellation.magic_edge.desc": "略微增加伤害并将其转化为魔法。",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "enchantment.ensorcellation.magic_edge.desc": "略微增加伤害，并将伤害转化为魔法伤害。",

```